src/util/inventory/__init__.py

```diff
@@ -226,7 +226,7 @@ def add_inhouse_inventory(inventory, fishbowl_inventory_report):
             inventory_key = FINISH_INVENTORY_KEY
         qty = int(float(qty))
         # Always use cost of zero to prioritize inhouse assignments
-        if inventory_key and inventory_key == CORE_INVENTORY_KEY:
+        if inventory_key:
             _add_to_inventory(inventory, inventory_key, part_num,
                               INHOUSE_VENDOR_KEY, qty, 0.0)
```

Comment:
There is a good opportunity here for an `else` block where we log the ones that don't fit the standard if the SQL query has the `LIKE` clauses removed.

Alternatively the inverse of the SQL query in `src/facade/fishbowl/fishbowl_facade.py:474-488` could be used to regularly print reports of non-compliant skus in fishbowl.

Either approach is a bit rough because it would include all non wheel part numbers. The best way would be to find only the inventory locations that have wheels and then do either approach limited to those locations. This ensure that the part numbers emitted *should* be wheels so there is less bs to sift through.

Reply:
@Sam58888 let me know if you would like to take care of this. You'll need to use sql and the fishbowl data module to figure out the location IDs. After that you'll modify the query I mentioned to filter `WHERE` the parts location is in the specified ranges or whatever it may be. 

The logging could be output to the FTP server, of which there are plenty of examples in the code. Alternatively, I recently set up cloudwatch logging for this so we could start with just logging to cloudwatch as a metric (number of malformed part numbers)